acceptance/current/.acceptance/acceptance-cookbook/.chef/config.rb

```diff
@@ -0,0 +1 @@
+cookbook_path File.join(File.dirname(__FILE__), '..', '..')
```

Comment:
without it chef doesn't recognize that the `.acceptance`dir the cookbooks dir.  Any advice were be appreciated.